### PR TITLE
feat(feedback): update empty state and onboarding panel

### PR DESCRIPTION
**Pull request by michellewzhang**
Updates the empty states/onboarding for beta, according to the [figma designs](https://www.figma.com/file/ga57zyLonVCSJIH3kOWLVA/Specs%3A-User-Feedback-v1?type=design&node-id=521-13457&mode=design&t=oMyArGtbU4qJYlPp-0)

No selected projects have feedback set up:
SCR-20231108-lrxo

At least 1 selected project has feedback set up but no results (either no feedback received or filters returned nothing): 
SCR-20231108-likh

Video demo:

https://github.com/getsentry/sentry/assets/56095982/0e61ab9b-42d2-45c8-8bd8-e26568a1bae4

Closes https://github.com/getsentry/team-replay/issues/270
Closes https://github.com/getsentry/team-replay/issues/271
Closes  https://github.com/getsentry/team-replay/issues/272

### CODE REVIEW

File: static/app/views/feedback/feedbackListPage.tsx [hunk 1]
```diff
@@ -33,14 +36,22 @@ export default function FeedbackListPage({}: Props) {
...
+                    <Container style={{gridArea: 'details'}}>
+                      <FeedbackItemLoader />
+                    </Container>
+                  </Fragment>
+                ) : null}
+                {!hasSetupOneFeedback && (
+                  <SetupContainer>
+                    <FeedbackSetupPanel />
+                  </SetupContainer>
+                )}
```
Comment (c298lee) [inline static/app/views/feedback/feedbackListPage.tsx:54]:
```suggestion
                ) :  <SetupContainer>
                    <FeedbackSetupPanel />
                  </SetupContainer>}
 
```

File: static/app/components/feedback/list/feedbackList.tsx [hunk 2]
```diff
@@ -28,6 +28,24 @@ const cellMeasurer = {
...
 };
 
+function Message({title, subtitle}: {subtitle: React.ReactNode; title: React.ReactNode}) {
+  return (
+    <Fragment>
+      <EmptyMessage>{title}</EmptyMessage>
+      <p>{subtitle}</p>
+    </Fragment>
+  );
+}
```
Comment (billyvg) [inline static/app/components/feedback/list/feedbackList.tsx:38]: Is this necessary? It seems to just be used in `NoFeedback`, you can inline it in there.

File: static/app/components/feedback/feedbackSetupPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+import newFeatureImg from 'sentry-images/spot/new-feature.svg';
+
+import {LinkButton} from 'sentry/components/button';
+import Panel from 'sentry/components/panels/panel';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+
+interface Props {
+  style?: CSSProperties;
```
Comment (billyvg) [inline static/app/components/feedback/feedbackSetupPanel.tsx:12]: Doesn't look like this prop gets used

**Review by billyvg — APPROVED** [review]
Envelope seems a tad small, but otherwise lgtm 
image

**Pull request merged**